docs/404.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Error 404
+hidden: false
+---
+
+<div markdown="1" class="404-page">
```

Comment:
**note (not blocking)**: if I add a third path segment, the styles won't load
test: https://demo2--mozilla-protocol.netlify.app/random/broken/link
<img width="825" alt="missing-styles" src="https://user-images.githubusercontent.com/19650432/223773464-62c6c45d-f5c2-4d02-bedb-41bf744b2810.png">

I think this is an internal Fractal issue, given we pass the theme css to their config: https://github.com/mozilla/protocol/blob/main/theme/index.js#L14
but it's relative, so too many path segments breaks it

2 path segments, css uses `themes` folder
<img width="538" alt="under-themes" src="https://user-images.githubusercontent.com/19650432/223775053-91c1f5cc-a91f-42ce-8d77-d96c03cdfa46.png">

3 path segments, css tries to find non-existent `random/themes` folder
<img width="481" alt="under-random-themes" src="https://user-images.githubusercontent.com/19650432/223775125-21566f15-5091-44b2-a0ea-f4f7259adc94.png">

I'd say this particular bug is a `wontfix`, just noting it for future reference